profiles/features/llvm/make.defaults

```diff
@@ -14,7 +14,8 @@ LD="ld.lld"
 # We don't set LDFLAGS here for linker, rtlib, etc because clang-common enforces
 # the needed bits for us and it confuses clang->gcc for stuff like Fortran,
 # see bug #908798.
-LDFLAGS="${LDFLAGS} -Wl,--as-needed"
+# FIXME: no idea where this belongs
+#LDFLAGS="${LDFLAGS} -Wl,--as-needed"
```

Comment:
I guess we can try filter -Wl,--as-needed out in LDFLAGS via profile.bashrc in the Darwin clang profiles, or we can just set LDFLAGS directly in make.defaults for now in our Darwin clang profiles (maybe let's do that for now?)

Reply:
I got another idea how to fix this. In my preliminary testing it works.